x-pack/filebeat/module/gsuite/saml/test/gsuite-saml-test.json.log-expected.json

```diff
@@ -0,0 +1,92 @@
+[
+    {
+        "@timestamp": "2020-10-02T15:00:00.000Z",
+        "client.as.number": 7922,
+        "client.as.organization.name": "Comcast Cable Communications, LLC",
+        "client.geo.city_name": "State College",
+        "client.geo.continent_name": "North America",
+        "client.geo.country_iso_code": "US",
+        "client.geo.location.lat": 40.7957,
+        "client.geo.location.lon": -77.8618,
+        "client.geo.region_iso_code": "US-PA",
+        "client.geo.region_name": "Pennsylvania",
+        "client.ip": "98.235.162.24",
+        "client.user.domain": "bar.com",
+        "client.user.email": "foo@bar.com",
+        "client.user.id": "1",
+        "client.user.name": "foo",
+        "event.action": "login_failure",
+        "event.category": "authentication",
+        "event.dataset": "gsuite.saml",
+        "event.id": "1",
+        "event.module": "gsuite",
+        "event.original": "{\"kind\":\"admin#reports#activity\",\"id\":{\"time\":\"2020-10-02T15:00:00Z\",\"uniqueQualifier\":1,\"applicationName\":\"saml\",\"customerId\":\"1\"},\"actor\":{\"callerType\":\"USER\",\"email\":\"foo@bar.com\",\"profileId\":1},\"ownerDomain\":\"elastic.com\",\"ipAddress\":\"98.235.162.24\",\"events\":{\"type\":\"login\",\"name\":\"login_failure\",\"parameters\":[{\"name\":\"application_name\",\"value\":\"app\"},{\"name\":\"failure_type\",\"value\":\"failure_app_not_configured_for_user\"},{\"name\":\"initiated_by\",\"value\":\"idp\"},{\"name\":\"orgunit_path\",\"value\":\"ounit\"},{\"name\":\"saml_second_level_status_code\",\"value\":\"400\"},{\"name\":\"saml_status_code\",\"value\":\"400\"}]}}",
+        "event.outcome": "failure",
+        "event.provider": "saml",
+        "event.type": "access",
+        "fileset.name": "saml",
+        "gsuite.actor.type": "USER",
+        "gsuite.event.type": "login",
+        "gsuite.kind": "admin#reports#activity",
+        "gsuite.organization.domain": "elastic.com",
+        "gsuite.saml.application_name": "app",
+        "gsuite.saml.failure_type": "failure_app_not_configured_for_user",
+        "gsuite.saml.initiated_by": "idp",
+        "gsuite.saml.orgunit_path": "ounit",
+        "gsuite.saml.second_level_status_code": "400",
+        "gsuite.saml.status_code": "400",
```

Comment:
would we ever want to do comparisons or ranges on the status codes?  Things like status_code > 400?  If so it might be a good idea for the status codes to be "longs" instead of "keywords"